articles/130_ros_time.md

```diff
@@ -140,6 +144,15 @@ The underlying datatypes will also provide ways to register notifications, howev
   <b>TODO:</b> Enumerate the <code>rcl</code> datastructures and methods here.
 </div>
 
+## Work Needing To Be Done (Apr 2017)
+
+- Decide where and when to autosubscribe to the `/clock` message. Built in to Node? A Node addon or service? Part of the first call to `rcl_node_init`?
+- Decide how to deal with multiple nodes in a single process that all update the one static time (or decide to ditch the static time methods).
+- Decide where to put the out-of-the-box clock publishers.
```

Comment:
This is driven by the use case, at least in ROS 1. For instance, rosbag would publish it when playing back data and gazebo will also publish it when running a simulation.